pkg/utils/util.go

```diff
@@ -131,7 +125,29 @@ func RunCommandString(command string, env []string, inFile *os.File, outFile *os
 	cmd.Stdout = outFile
 	cmd.Stderr = errFile
 
-	if err := cmd.Run(); err != nil {
+	return execCommand(cmd)
+}
+
+func execCommand(cmd *exec.Cmd) (int, error) {
+	if err := cmd.Start(); err != nil {
+		if exitError, ok := err.(*exec.ExitError); ok {
+			return exitError.ExitCode(), err
+		}
+
+		return 1, err
+	}
+
+	// interrupts will be sent to the entire process group (parent and child processes)
+	// catch + ignore the parent's interrupt so that we use the exit code from the child
+	channel := make(chan os.Signal, 1)
+	signal.Notify(channel, os.Interrupt, syscall.SIGHUP, syscall.SIGQUIT, syscall.SIGTERM)
+	go func() {
+		s := <-channel
+		// do nothing; the child process will have also received the signal
+		LogDebug(fmt.Sprintf("Process received %s", s.String()))
+	}()
```

Comment:
Is this a windows thing where both the parent and child receive the signal?

Reply:
It's not Windows specific. The child process will be in the same process group as the parent and so it will receive the same signals.

Reply:
Hmm how come this isn't an issue for Mac or linux users then? Is Window's doing something different?

Reply:
Windows's `cmd` displays a nonsense "Terminate batch job" prompt that keeps the child process from dying. Before the user can respond the parent process has already been killed.